### PR TITLE
Initial fit of HI and LO coils, without tests or power draw method.

### PR DESCRIPTION
**Pull request by e-wai**
Based on Clarke's work in alt_gain.py and Micah's data [here](https://uwmidsun.atlassian.net/wiki/spaces/MECH/pages/1628012551/Interpreting+the+Graphs+From+Nomura).

+ No tests or power draw so far (for future tickets)
+ I fit it pretty visually... Split into a log func (also tried out power, but didn't work quite as nicely) and the linear section. These aren't super accurate, so I was leaning towards using residuals to keep track of error.

LO Coil
HI Coil

### CODE REVIEW

**Review by ufshk — COMMENTED** [review]
So, I'm not sure what to add. I think the code is fine?

**Comment by e-wai** [comment]
I should run this by Micah Black to make sure I'm representing the graphs correctly. 

I'm more trying to figure out if the best way to get the most accurate values is this way?

File: optimization/motor_efficiency.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        self.m = round((self.efficiency[-1] - self.efficiency[4]) \
+            / (self.torque[-1] - self.torque[4]), 8)
+        self.b = self.efficiency[-1] - self.m * self.torque[-1]
+
+        np.append(self.res,
+            [self.efficiency[point] 
+            - self.linear_func(self.torque[point], self.m, self.b)
+            for point in range(len(self.torque[6:]))])
+
+    def graph(self):
```
Comment (e-wai) [inline optimization/motor_efficiency.py:41]: add axis labels @e-wai

**Review by mdshiozaki — APPROVED** [review]
Checks out to me

**Pull request closed without merging**